web/views.py

```
@@ -6924,3 +6940,475 @@ def delete_post(request, post_id):
+            response = ai_service.get_response(message)
+            
+            return JsonResponse({'response': response})
+        except json.JSONDecodeError:
+            return JsonResponse({'error': 'Invalid JSON'}, status=400)
+        except Exception as e:
+            return JsonResponse({'error': str(e)}, status=500)
+    else:
+        return JsonResponse({'error': 'Method not allowed'}, status=405)
+
+@require_http_methods(["GET", "POST"])
+def ai_chat(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `ai_chat`

(ANN201)